lua-zlib-scm-0.rockspec

```diff
@@ -0,0 +1,42 @@
+package = "lua-zlib"
+version = "scm-0"
+source = {
+   url = "git://github.com/brimworks/lua-zlib.git",
+   branch = "master",
+}
+description = {
+   summary = "Simple streaming interface to zlib for Lua.",
+   detailed = [[
+      Simple streaming interface to zlib for Lua.
+      Consists of two functions: inflate and deflate.
+      Both functions return "stream functions" (takes a buffer of input and returns a buffer of output).
+      This project is hosted on github.
+   ]],
+   homepage = "https://github.com/brimworks/lua-zlib",
+   license = "MIT"
+}
+dependencies = {
+   "lua >= 5.1"
```

Comment:
I'm reluctant to remove the upper bound since minor versions of Lua often change the C API in backwards incompatible ways.

Reply:
True, Lua's release schedule and versioning is less than stellar. That being said 5.3.5 to 5.4 didn't seem to break much this time around. I got kind of frustrated having to track down a bunch of LuaRock maintainers to bump upper bounds for things that I knew worked fine in my project if I patched out the restrictions. I'd prefer running the risk of running into the breakage myself on new versions of Lua than being guaranteed breakage because module authors pre-emptively blocked it.